core/src/main/java/org/mule/api/context/notification/FlowStackElement.java

```diff
@@ -13,17 +13,17 @@
  * 
  * @since 3.8.0
  */
-public interface FlowStackElement extends Serializable, Cloneable
+public interface FlowStackElement extends Serializable
```

Comment:
These are so that clients of the api don't have access to the methods that change the state of the flowstack.